dist/openshift/cincinnati-deployment.yaml

```diff
@@ -189,7 +189,7 @@ objects:
           metadata:
             serverAddress: http://prometheus-app-sre.openshift-customer-monitoring.svc.cluster.local:9090
             metricName: cincinnati_policy_engine_graph_incoming_requests_rate
-            threshold: "${{PE_REQ_AVG}}"
+            threshold: "${PE_REQ_AVG}"
```

Comment:
Would the following be preferable for consistency with how we use e.g. MAX_REPLICAS/MIN_REPLICAS above on L185-186?

```suggestion
            threshold: ${{PE_REQ_AVG}}
```

Reply:
:+1: